serialization.md

```diff
@@ -472,4 +472,42 @@ The resulting Tar archive for `f60c56784b83` has the following entries:
 /etc/.wh.my-app-config
 ```
 
+In addition to expressing that a single entry should be removed from a lower layer, layers may remove all of the children using an opaque whiteout entry.
```

Comment:
Symlinks don't have children, by definition, but feel free to test it out.

From the paragraph above, it is fairly clear we are talking about directories. If you insist, I can add something to clarify it.